desktop/lib/menu/editor-context-menu.js

```diff
@@ -0,0 +1,87 @@
+
+/**
+ * Original library:
+ * Author: Jeff Wear, Mixmax, Inc
+ * License: MIT
+ * From: https://github.com/mixmaxhq/electron-editor-context-menu
+ */
+
+var noop = function(){};
+var cloneDeep = require('lodash.clonedeep');
+var BrowserWindow = require('electron').BrowserWindow;
+var Menu = require('electron').Menu;
+var platform = require( '../platform' );
+
+var DEFAULT_MAIN_MENU = [{
+  label: 'Cut',
+  role: 'cut'
+}, {
+  label: 'Copy',
+  role: 'copy'
+}, {
+  label: 'Paste',
+  role: 'paste'
+}, {
+  label: 'Select All',
+  role: 'selectall'
```

Comment:
Minor: we should fix the indents in this file (guessing the spaces are from the original lib).